src/scriptwriter/scenarios/ScenariosHandler.js

```diff
@@ -110,4 +113,14 @@ export default class ScenariosHandler {
 
         return scenario;
     }
+
+    /**
+     * Loads a start url for scenario from specified url paths
+     * @returns {string} start url
+     */
+    _getRandomScenarioStartUrl() {
+        let scenarios = this._config.urlPaths;
+
+        return scenarios[Math.floor(Math.random() * scenarios.length)];
```

Comment:
I would rather iterate through the urlPaths one by one than selecting random value.

Reply:
I would also return `this._config.url + path` and work with full url later on, since we are calling it `startUrl`, not `startUrlPath`

Reply:
Newly return `this._config.url + path`